<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapManagerEdit</class>
 <widget class="QFrame" name="MapManagerEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_lab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>220</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lab_lab_head">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>64</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Метка</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_lab_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_lab_apply">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_lab_add">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>новая метка</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_lab_rem">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>удалить метку</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_lab_font_size">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>размер шрифта</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_lab_z_index">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Z индекс</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
     <widget class="QLabel" name="lab_color_f">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>цвет текста</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_color_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>цвет фона</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_color_f">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_color_b">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_cab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lab_line_head">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>64</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Кабель</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_line_apply">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_line_size">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>толщина линии</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_line_z_index">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Z индекс</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_line_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>тип линии</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>347</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_line_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pb_line_apply</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_line_z_index</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pb_line_apply</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_line_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pb_line_apply</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
